docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic.TensorFlow
+{
+    class TextClassification
+    {
+        public const int MaxSentenceLenth = 600;
+        /// <summary>
+        /// Example use of the TensorFlow sentiment classification model.
+        /// Download the model from 
+        /// https://github.com/dotnet/machinelearning-testdata/blob/master/Microsoft.ML.TensorFlow.TestModels/sentiment_model
+        /// The model is in 'SavedModel' format. For further explanation on how was the `sentiment_model` created 
+        /// c.f. https://github.com/dotnet/machinelearning-testdata/blob/master/Microsoft.ML.TensorFlow.TestModels/sentiment_model/README.md
+        /// </summary>
+        public static void ScoringWithTextClassificationModelSample()
+        {
+            string modelLocation = @"sentiment_model";
+
+            var mlContext = new MLContext(seed: 1, conc: 1);
```

Comment:
>seed: 1, conc: 1 [](start = 42, length = 16)

let's use the simplest version, in general, for those samples. No need to set the seed and conc